start_admin.sh

```diff
@@ -127,10 +171,10 @@ fi
 
 chown -R "${LOCAL_USER}:" "${USER_SSH_DIR}"
 
-# If there were no successful auth methods, then users cannot authenticate
+# If there were no available SSH auth methods, then users cannot connect via the SSH daemon
 if [[ "${available_auth_methods}" -eq 0 ]]; then
```

Comment:
```suggestion
if [[ "${available_ssh_methods}" -eq 0 ]]; then
```
Since we are only counting ssh authentication methods, it would probably be best to rename this to reduce confusion.